packages/node_modules/@ciscospark/widget-recents/src/components/space-item/index.js

```diff
@@ -119,9 +119,10 @@ export default function SpaceItem({ // eslint-disable-line complexity
         {lastActivityTime}
       </div>
       {
-        isDirectSpace && hasCallSupport &&
+        hasCallSupport &&
         <div className={classNames(`space-actions`, styles.actions)}>
           <Button
+            accessibilityLabel={`Call Space`}
```

Comment:
Do we need to start putting these labels into `intl` wrappers so that eventually they can be translated?

Reply:
Yep. I created a ticket to fix all of these. https://voxeolabs.atlassian.net/browse/SSDK-1464